steamapi/store.py

```diff
@@ -0,0 +1,53 @@
+__author__ = 'andrew'
+
+import uuid
+from .core import APIConnection
+
+
+class SteamStore(object):
```

Comment:
Since this is used for the in-game item stores, rename this to something more specific, like `SteamIngameStore`. (I plan to add an API for the main store someday)